ci/run_tests.sh

```diff
@@ -1,4 +1,15 @@
 #!/bin/sh
 set -evx
 
-chef-server-ctl test -J $WORKSPACE/pedant.xml --all
+sudo chef-server-ctl test -J $WORKSPACE/pedant.xml --all
+
+if [ $OMNIBUS_FIPS_MODE == "true" ]
```

Comment:
Based on discussion in Slack, let's only run the second round of test if the first pass succeeds.  That will prevent overwriting/losing the pedant.xml output that jenkins uses to report on the test results.  Let's add a FIXME note here to look into what it'd take to have jenkins support multiple test outputs (maybe a simple opscode-ci update), but that shouldn't block merging.